Kickstarter-iOS/Views/Controllers/BetaToolsViewController.swift

```diff
@@ -131,6 +131,11 @@ internal final class BetaToolsViewController: UIViewController {
       UIAlertAction.init(title: "Cancel", style: .cancel)
     )
 
+    if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
Just a thought, we could also use `AppEnvironment.current.device`, which would allow us to mock devices and test this logic. Looking at it more carefully, there's probably a case for adding this logic for the view model (right now `showEnvironmentActionSheet` is being called directly from the tap gesture handler) and writing a test for it.

Reply:
After digging a little more, I noticed that the `self.dismiss(animated:)` function is not actually necessary, because the AppDelegate will set the destination ViewController once the `.ksr_languageChanged` notification is fired. And the completion block wasn't being called (I just wonder since when this is happening). 

Also, the `UIDevice.current.userInterfaceIdiom == .pad` is the convention we use on several places of the app just to set the UIAlertController's modal presentation type. In this particular case, I think @dusi 's solution to move the logic to an extension might be quicker and more efficient. What do you think?